components/support/sync-telemetry/pings.yaml

```diff
@@ -4,6 +4,23 @@
 
 $schema: moz://mozilla.org/schemas/glean/pings/1-0-0
 
+sync:
+  description: >
+    A summary ping, sent every time a sync is performed. During each Sync one or more data types
+    could be synchronized, depending on which data types user configured to sync. Alongside with 'sync' ping
+    one or more individual data type specific pings will be sent. For example, if history and bookmarks
+    data types are configured to be synchronized, the following pings will be sent: 'sync', 'history-sync'
+    and 'bookmarks-sync'. Alternatively, if only history is configured to be synchronized then 'sync' and 'history-sync'
+    pings will be sent. In case of a "global failure" where none of the data type syncs could even start,
+    e.g. device is offline, only the 'sync' ping will be sent.
+    This ping doesn't include the `client_id` because it reports a hashed version of the user's Firefox Account ID.
+  include_client_id: false
+  bugs:
+    - https://github.com/mozilla-mobile/android-components/issues/5371
+  notification_emails:
+    - sync-core@mozilla.com
+  data_reviews:
+    - TODO
```

Comment:
Just a reminder to update the link to the data review **response** after you have one. Here and in the other yaml files.

Reply:
Yup, waiting for that, hence a `TODO` :)